### DISCUSSION TITLE
Is there any `autobump` documentation?

### DISCUSSION THREAD

**Discussion by yurikoles** [Homebrew/discussions#5468] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.8-12-g3aeef5a
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3aeef5aef2030e200d3b80273e3e79cf74b240c1
Last commit: 2 hours ago
Core tap HEAD: 5a1b6edd6912ec627ec86e48a17c140a982f0300
Core tap last commit: 12 minutes ago
Core tap JSON: 01 Jul 09:15 UTC
Core cask tap HEAD: d1bf02c3a2b677a1944d89d43e7d698b6dbca8ac
Core cask tap last commit: 14 minutes ago
Core cask tap JSON: 01 Jul 09:15 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: ["--no-quarantine"]
HOMEBREW_DEVELOPER: set
HOMEBREW_EDITOR: subl -w
HOMEBREW_FORCE_BREWED_CURL: set
HOMEBREW_FORCE_BREWED_GIT: set
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.3 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 16.0.0 build 1600
Git: 2.45.2 => /opt/homebrew/opt/git/bin/git
Curl: 8.8.0 => /opt/homebrew/opt/curl/bin/curl
macOS: 14.5-arm64
CLT: 16.0.0.0.1.1719078471
Xcode: 16.0 => /Applications/Xcode_16_beta_2.app/Contents/Developer
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1

Warning: Putting non-prefixed coreutils in your path can cause GMP builds to fail.
```

### Description of issue

I was unable to find `autobump` documentation. In particular, my question is there any requirements for formula or cask to qualify for it? I see that now project uses explicit allowlist for formulas and cask that is being autobumped. Is there any motivation behind this decision?

**Comment by SMillerDev** [comment]
There is no explicit documentation for it, but we're adding it for formula that are not giving a lot of issues for updates.

> I see that now project uses explicit allowlist for formulas and cask that is being autobumped. Is there any motivation behind this decision?

We started out small, which makes an allowlist easier. We could probably make a blocklist by now, but nobody has done so yet.

**Comment by yurikoles** [comment]
I'm thinking about `autoautobump`, pardon me for this tautology, that will once a day check formulae and casks history. And If last `N`, e.g. 3, version upgrades were _easy_, i.e. just bump of artefacts, resources, and their checksums that can be discovered by `livecheck`, then this app should be added to `autobump` list or removed otherwise.

Would you consider such a pull request?
